MMA Removal Utility/README.md

```diff
@@ -105,7 +105,7 @@ Update-AzTSMMARemovalUtilityDiscoveryTrigger `
 
 ```
 ### Schedule or Trigger Removal phase
-Removal phase is disabled by default. Users need to enable/schedule it after validating the inventory of VMs list prepared in the Discovery phase. Command **Update-AzTSMMARemovalUtilityRemovalTrigger** can be used to:
+Removal phase is disabled by default. Users need to enable/schedule it after validating the inventory of resource list prepared in the Discovery phase. Command **Update-AzTSMMARemovalUtilityRemovalTrigger** can be used to:
```

Comment:
NIT: inventory of resource list --> inventory of resources

Reply:
Fixed